atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -134,6 +145,33 @@ def adjust_for_autorebuild(self):
         if self.compose_ids:
             self.log.info('Autorebuild detected: Ignoring compose_ids plugin parameter')
             self.compose_ids = tuple()
+            self.all_compose_ids = []
+
+    def adjust_for_inherit(self):
+        build_info = self.plugin_result[BASE_IMAGE_KOJI_BUILD]
```

Comment:
actually I'll check if plugin_result is None, because if it runs and finishes that key will be always defined, but might have None value, which will handle following tries which are getting values